src/Components/Button/index.tsx

```diff
@@ -1,33 +1,40 @@
-import React, { FC } from 'react'
+import React, { FC, CSSProperties } from 'react'
 import { Button } from 'react-bootstrap'
 
-import { generateRenderProps } from '../../Utils/helpers'
+import { generateRenderProps, generateThemeColors } from '../../Utils/helpers'
+import { Colors, colorSchemeLight, Theme } from '../../Constants/Types/theme.types'
 
-import styles from './Button.module.scss'
+// import styles from './Button.module.scss'
+import './Button.scss'
 
 interface Props {
+  theme: Theme
   text?: string
   variant?: string
   className?: string
-  style?: { color: string, borderColor: string }
+  style?: CSSProperties
```

Comment:
add "colors" prop which is 
```
type ColorScheme = {}

export type Colors = {
  [theme in Theme]?: ColorScheme
}
```
You did it right in types so we should pass {DARK: colorSchemeDark, LIGHT: colorSchemeLight}
so then for example backgroundColor = props.colors[theme]. backgroundColor
It allows us to change theme from parent component to switch colors.